scripts/azure/staging_query_submit.sh

```
@@ -0,0 +1,33 @@
+# Manual spark submission of staging query for debugging.
```

Shellcheck (0.11.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)